packages/cbioportal-ts-api-client/src/generated/CBioPortalAPIInternal.ts

```diff
@@ -5537,6 +5553,14 @@ export default class CBioPortalAPIInternal {
             'sampleMolecularIdentifiers0MolecularProfileId' ? : string,
             'sampleMolecularIdentifiers0SampleId' ? : string,
             'structuralVariantFilter': StructuralVariantFilter,
+            'structuralVariantQueries0Gene1ContainingOnlyOneIdentifierOrSpecialValue' ? : boolean,
```

Comment:
seems kinda strange that there's index in this property name?

Reply:
Ah yes, this required running `yarn updateAPI` against the correct backend branch. It is fixed now.

Reply:
This fixed with merge of the backend PR.